### ISSUE TITLE
The matrix U and V in implementation

### ISSUE THREAD

**Issue by RongKaiWeskerMA** [luyao777/HBP-Pytorch#11]
Thanks for implementing the paper! I am confused with the matrix U and V. These two matrices in the paper seem to be two different matrices. But in the implementation, they are initilialized as the same matrix as:
self.bilinear_proj = torch.nn.Sequential(torch.nn.Conv2d(512,8192,kernel_size=1,bias=False),
                                        torch.nn.BatchNorm2d(8192),
                                        torch.nn.ReLU(inplace=True))

proj_1 = self.bilinear_proj(conv1)
proj_2 = self.bilinear_proj(conv2)

**Comment by HeimingX** [comment]
I am also curious about the implementation of this part. Look forward the feedback from author @luyao777

**Comment by luyao777** [comment]
Calling this function repeatedly creates different instances.
